src/app/qgsidentifyresultsdialog.cpp

```diff
@@ -1130,6 +1134,12 @@ void QgsIdentifyResultsDialog::addFeature( QgsMeshLayer *layer,
     connect( layer, &QgsMapLayer::crsChanged, this, &QgsIdentifyResultsDialog::layerDestroyed );
   }
 
+  // for Mesh layers it looks like it is best to hide the 'Geometry' feature, but keep the actual 'derived' attributes
+  if ( label == "Geometry" && QgsSettings().value( QStringLiteral( "/Map/hideDerivedAttributes" ), false ).toBool() )
```

Comment:
This won't work for non-english installs. 
```suggestion
  if ( label == tr( "Geometry" ) && QgsSettings().value( QStringLiteral( "/Map/hideDerivedAttributes" ), false ).toBool() )
```

Reply:
Ah thanks. Yes I was wondering if such words were translated or not after my commit...